caffe2/mobile/contrib/arm-compute/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+if(USE_ARM_COMPUTE AND ANDROID)
```

Comment:
Typically we just disable options on unsupported platforms, e.g. see https://github.com/caffe2/caffe2/blob/1f9e79f920a22aed1a370bf5faec1ce5e1ef73d7/cmake/Dependencies.cmake#L460-L465